README.md

```diff
@@ -36,7 +36,7 @@ If the Terminal tells you that it can't find it, you'll have to get it. Go to ht
 **Get your development and test databases ready:**
 
     foreman run rake db:setup
-    foreman run rake db:migrate RAILS_ENV=test
+    foreman run rake db:create db:migrate RAILS_ENV=test
```

Comment:
Doesn't `db:setup` runs `db:schema:load, db:seed` ? Then it would make sense to:

```
foreman run rake db:create:all 
foreman run rake db:setup
foreman run rake db:setup RAILS_ENV=test
```
Really need to make sure `schema.rb` is updated and versioned control with this :) is it ok with you ?